src/pages/files/index.tsx

```diff
@@ -30,128 +31,138 @@ const useStyles = makeStyles((theme: Theme) =>
+  const [uploadingFile, setUploadingFile] = useState(false)
 
-    const getFile = () => {
-      setLoadingSearch(true)
-        beeApi.files.downloadFile(searchInput)
-        .then(res => {
-            setSearchResult(res)
-            
-            const downloadUrl = window.URL.createObjectURL(new Blob([res.data]));
-            const link = document.createElement('a');
-            link.href = downloadUrl;
-            link.setAttribute('download', 'file.zip'); //any other extension
-            document.body.appendChild(link);
-            link.click();
-            link.remove();
-        })
-        .catch(error => {
-        })
-        .finally(() => {
-          setLoadingSearch(false)
-        })
-    }
+  const getFile = () => {
+    setLoadingSearch(true)
+    beeApi.files
+      .downloadFile(searchInput)
+      .then(res => {
+        setSearchResult(res)
 
-    const uploadFile = () => {
-      setUploadingFile(true)
-        beeApi.files.uploadFile(files[0])
-        .then(hash => {
-            setUploadReference(hash)
-            setFiles([])
-        })
-        .catch(error => {
-        })
-        .finally(() => {
-          setUploadingFile(false)
-        })
-    }
+        const downloadUrl = window.URL.createObjectURL(new Blob([res.data]))
+        const link = document.createElement('a')
+        link.href = downloadUrl
+        link.setAttribute('download', 'file.zip') //any other extension
+        document.body.appendChild(link)
+        link.click()
+        link.remove()
+      })
+      .catch(() => {
+        // FIXME: handle the error
+      })
+      .finally(() => {
+        setLoadingSearch(false)
+      })
+  }
 
-    const handleChange = (files: any) => {
-      if (files) {
-        setFiles(files)
-      }
+  const uploadFile = () => {
+    setUploadingFile(true)
+    beeApi.files
+      .uploadFile(files[0])
+      .then(hash => {
+        setUploadReference(hash)
+        setFiles([])
+      })
+      .catch(() => {
+        // FIXME: handle the error
+      })
+      .finally(() => {
+        setUploadingFile(false)
+      })
+  }
+
+  const handleChange = (files?: File[]) => {
+    if (files) {
+      setFiles(files)
     }
+  }
 
-    return (
-        <div>
-            {props.nodeHealth?.status === 'ok' && props.health ?
-            <Container maxWidth="sm">
-              <div style={{marginBottom: '7px'}}>
-                <Button color="primary" style={{marginRight: '7px'}} onClick={() => setInputMode('browse')}>Browse</Button>
-                <Button color="primary" onClick={() => setInputMode('upload')}>Upload</Button>
-              </div>
-              {inputMode === 'browse' ? 
-              <Paper component="form" className={classes.root}> 
-                  <InputBase
+  return (
+    <div>
+      {
+        // FIXME: this should be broken up
+        /* eslint-disable no-nested-ternary */
+        nodeHealth?.status === 'ok' && health ? (
```

Comment:
Seems like a duplicate of a previous comment. Would be interesting to investigate if the duplication is worth eliminating.